presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -94,6 +94,7 @@
     TOO_MANY_TASK_FAILED(0x0001_001B, INTERNAL_ERROR),
     INDEX_LOADER_TIMEOUT(0x0001_001C, INTERNAL_ERROR),
     EXCEEDED_TASK_UPDATE_SIZE_LIMIT(0x0001_001D, INTERNAL_ERROR),
+    SCHEDULE_NOT_SUPPORTED(0x0001_001E, INTERNAL_ERROR),
```

Comment:
nit: What about `SPLIT_SCHEDULE_STRATEGY_NOT_SUPPORTED`. "schedule" itself is an umbrella word so might want to be more specific in error code :)